velox/exec/tests/PlanNodeToSummaryStringTest.cpp

```diff
@@ -150,6 +150,48 @@ TEST_F(PlanNodeToSummaryStringTest, expressions) {
           {.project = {.maxProjections = 3, .maxDereferences = 2}}));
 }
 
+// Below test is to verify that the summary string is truncated when
+// nodeHeaderOnly is set to true.
+TEST_F(PlanNodeToSummaryStringTest, DisplaysNodeHeadersOnly) {
```

Comment:
DisplaysNodeHeadersOnly -> nodeHeaderOnly

test names should start with lower case